.datalad/config

```diff
@@ -0,0 +1,2 @@
+[datalad "dataset"]
+	id = 3db13625-3f89-41cc-a885-83adbe45d420
```

Comment:
how was this id generated?

Reply:
datalad create -f in the repo

Reply:
sorry it's unclear to me-- if you create a datalad in the repos and you also have to push the datalad info (eg .datalad folder or something like that?)

Reply:
shouldn't we use our conventional way of storing data in the lab with git-annex (instead of datalad, which adds a layer of complexity and is not documented in our [lab intranet](https://intranet.neuro.polymtl.ca/computing-resources/data/git-datasets.html)), tagging @kousu and @mguaypaq

is there a conversation that led to this decision for using datalad? maybe on slack or during our meeting, or on any GH repository?

Reply:
mentioned here https://github.com/ivadomed/ivadomed/issues/167